src/backend/graphics.rs

```diff
@@ -164,12 +172,28 @@ fn crop_context(context: Context, rect: Rect) -> Context {
             h = 0;
         } else {
             // If there is some intersection, calculate the overlapping rect.
-            let (a_l, a_r, a_b, a_t) = (x, x+w, y, y+h);
-            let (b_l, b_r, b_b, b_t) = (rect.x, rect.x+rect.w, rect.y, rect.y+rect.h);
-            let l = if a_l > b_l { a_l } else { b_l };
-            let r = if a_r < b_r { a_r } else { b_r };
-            let b = if a_b > b_b { a_b } else { b_b };
-            let t = if a_t < b_t { a_t } else { b_t };
+            let (a_l, a_r, a_b, a_t) = (x, x + w, y, y + h);
+            let (b_l, b_r, b_b, b_t) = (rect.x, rect.x + rect.w, rect.y, rect.y + rect.h);
+            let l = if a_l > b_l {
+                a_l
+            } else {
+                b_l
+            };
+            let r = if a_r < b_r {
+                a_r
+            } else {
+                b_r
+            };
+            let b = if a_b > b_b {
+                a_b
+            } else {
+                b_b
+            };
+            let t = if a_t < b_t {
+                a_t
+            } else {
+                b_t
+            };
```

Comment:
Hmmm I think I much prefer these in their previous state, as the symmetry is a bit more readable (I noticed this in a couple of other places also). Do you know if there's some kind of rustfmt config we can add to the repository root that will stop rustfmt from doing this?